Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeItem.cs

```diff
@@ -67,7 +68,9 @@ protected override void OnExpanded(EventArgs args)
             {
                 Width = _normalModeWidth;
                 VisualStateManager.GoToState(this, "Expanded", true);
-                AutomationProperties.SetName(this, "Expanded");
+                var name = StringExtensions.GetLocalized("Microsoft.Toolkit.Uwp.UI.Controls/Resources", "WindowsCommunityToolkit_BladeView_ExpandButton_Collapsed");
+                System.Diagnostics.Debug.Print(name);
+                AutomationProperties.SetName(this.GetTemplateChild("EnlargeButton"), name);
```

Comment:
'EnlargeButton' is part of the template already, but it could also be null, so we need to check that here in the `_enlargeButton` member.

Reply:
I just uploaded this modification in the last commit.